### PR TITLE
Dev yolov7

### PR DESCRIPTION
**Pull request by MarcBresson**
add yolov7 pose tracker to the project and remove alphapose entirely

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -23,7 +23,9 @@ def get_version(rel_path: str) -> str:
 install_requires = [
     "requests",
     "cython",
-    "deep-sort-realtime"
+    "deep-sort-realtime",
+    "torch",
+    "TACTUS - yolov7",
```
Comment (Alcharyx) [inline setup.py]: No comma?

File: tactus_data/utils/yolov7.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+from torchvision import transforms
+import cv2
+from torch.hub import download_url_to_file
+from tactus_yolov7 import Yolov7
```
Comment (Alcharyx) [inline tactus_data/utils/yolov7.py:4]: Not same name in setup.py also check for deep_sort_real_time in retracker.py (not this commit but still)

File: README.md [hunk 3]
```diff
@@ -15,23 +15,16 @@
...
 
-Downoad [AlphaPose](https://github.com/MVIG-SJTU/AlphaPose) on your machine with the follwing command:
+Download and install [Yolov7](https://github.com/Cranfield-GDP3/yolov7) on your machine with the follwing command:
 
 ```bash
-git clone https://github.com/MVIG-SJTU/AlphaPose.git
-cd AlphaPose
+git clone https://github.com/Cranfield-GDP3/yolov7
+cd yolov7
 python -m pip install -e .
```
Comment (MarcBresson) [inline README.md]: directly pip install the link

File: setup.py (hunk 1 above)
Comment (MarcBresson) [inline setup.py]: try putting the github link

**Pull request merged**